api/src/pcapi/core/users/eligibility_api.py

```diff
@@ -65,11 +110,15 @@ def get_eligibility_at_date(
     if not age:
         return None
 
-    if age in constants.ELIGIBILITY_UNDERAGE_RANGE:
-        return users_models.EligibilityType.UNDERAGE
-    # If the user is older than 18 in UTC timezone, we consider them eligible until they reach eligibility_end
-    if constants.ELIGIBILITY_AGE_18 <= age and specified_datetime < eligibility_end:  # type: ignore[operator]
-        return users_models.EligibilityType.AGE18
+    if specified_datetime < CREDIT_V3_DECREE_DATETIME or not FeatureToggle.WIP_ENABLE_CREDIT_V3.is_active():
+        if age in constants.ELIGIBILITY_UNDERAGE_RANGE:
+            return users_models.EligibilityType.UNDERAGE
+        # If the user is older than 18 in UTC timezone, we consider them eligible until they reach eligibility_end
+        if constants.ELIGIBILITY_AGE_18 <= age and specified_datetime < eligibility_end:  # type: ignore[operator]
+            return users_models.EligibilityType.AGE18
+
+    if FeatureToggle.WIP_ENABLE_CREDIT_V3.is_active() and age >= 17:
```

Comment:
On peut être exigeant ici, vu qu'on est à la date spécifiée
```suggestion
    if FeatureToggle.WIP_ENABLE_CREDIT_V3.is_active() and 17 <= age <= 18:

Reply:
Vu ensemble : ok